vug_pilot2/VehicleConfigParams.yaml

```diff
@@ -0,0 +1,145 @@
+# Defines the ros parameters which define the characteristics of this host vehicle configuration
+# The value type field is used to indicate how the field should be set.
+# Should it be treated as a measured value (Measured) or a desired behavior constraint (Desired)
+use_sim_time: true
```

Comment:
Just added this

Reply:
@adev4a 
please remember to add to `ail_vru_uc1_scenario` folder and set it `false` otherwise cdasim would break down. I would even add to xil_cloud and telematics ones as well.